### DISCUSSION TITLE
Brownie circular import error

### DISCUSSION THREAD

**Discussion by naman1402** [smartcontractkit/full-blockchain-solidity-course-py#1862] [Q&A]
I am learning about brownie framework, but I suddenly ran into this issues ? 

`ImportError: cannot import name 'Accounts' from partially initialized module 'brownie.network.account' (most likely due to a circular import) (D:\lib\site-packages\brownie\network\account.py)`

Can someone please help me out asap ?

**Comment by Niferu** [comment]
Hello @NamanMohnani 

Show us how you are trying to import it as you should import accounts like below:

```py
from brownie import accounts
```

Remember that imports are case sensitive

**Comment by naman1402** [comment]
Hello @Neftyr Thank you for reaching out.

for better explanation of my problem, let's say
```
from brownie import Lottery, accounts, config, network
from web3 import Web3


def test_get_entrance_fee():
    account1 = accounts[0]
    lottery = Lottery.deploy(
        config["networks"][network.show_active()]["eth_usd_price_feed"],
        {"from": account1},
    )
    assert lottery.GET_ENTRANCE_FEE() > Web3.toWei(0.008, "ether")
    assert lottery.GET_ENTRANCE_FEE() < Web3.toWei(0.062, "ether")
    # =============== test session starts ================
    # platform win32 -- Python 3.10.6, pytest-6.2.5, py-1.11.0, pluggy-1.0.0
    # rootdir: E:\College shit\coding\SmartContract-Lottery-Brownie
    # plugins: eth-brownie-1.19.3, hypothesis-6.27.3, forked-1.4.0, xdist-1.34.0, web3-5.31.3
    # collected 1 item
    # This version of µWS is not compatible with your Node.js build:

    # Error: Cannot find module './uws_win32_x64_111.node'
    # Falling back to a NodeJS implementation; performance may be degraded.

    # Launching 'ganache-cli.cmd --chain.vmErrorsOnRPCResponse true --wallet.totalAccounts 10 --fork.url https://eth-mainnet.g.alchemy.com/v2/5kE6AiZirrUG6CPUphZ4QZ4E5MSsywYW --wallet.mnemonic brownie --server.port 7545 --hardfork istanbul'...

    # tests\test_lottery.py .                       [100%]

    # ================ 1 passed in 24.80s ================
    # Terminating local RPC client...``

```
And on running 
`$ brownie test`

I get, 

INFO: Could not find files for the given pattern(s)
from brownie.network.account import Accounts
ImportError: cannot import name 'Accounts' from partially initialized module 'brownie.network.account' (most likely due to a circular import) 

And even when I try 
`brownie networks --list`
I get the same issue

**Reply by Niferu** [reply]
Hi @NamanMohnani 

Seems like your brownie is installed incorrectly. What interpreter are you using? You can also try to run it with custom interpreter as evm for brownie. Also check if your python version is lower than 3.10 as this and all above version will mess with brownie.

**Reply by cromewar** [reply]
I'm on the same page as @Neftyr, I think you brownie installation is corrupted. I strongly suggest to create new python env, and install eth-brownie on it, using pip.

**Comment by naman1402** [comment]
@cromewar @Neftyr then do you suggest me too delete my Python and brownie and reinstall it. Also, I am currently learning hardhat so is hardhat a good option as an alternative to brownie ?

And @Neftyr my python version is 3.10.6

**Comment by Niferu** [comment]
Hello @NamanMohnani 

Hardhat is using different programming language, which is JavaScript. If you are following this course based on python you should use brownie. Just go to this link and follow instruction carefully https://chain.link/bootcamp/brownie-setup-instructions

Especially below part if you are using windows as this can cause your problems even if you dont see C++ errors (you should install this tool pack):
![image](https://user-images.githubusercontent.com/110537615/231897156-cd5e5738-2be8-4a4c-b72f-f5c78807966e.png)

Also check your interpreter as it should look like below:
![image](https://user-images.githubusercontent.com/110537615/231897233-a8efdf4b-93d4-454a-8e12-0f0262edbddb.png)

If it is not show us what you are having in your interpreter list

**Reply by cromewar** [reply]
Does this make sense to you @NamanMohnani ?
